explorer/explorer.go

```diff
@@ -218,6 +236,8 @@ type explorerUI struct {
 	Mux              *chi.Mux
 	blockData        explorerDataSourceLite
 	explorerSource   explorerDataSource
+	onChainSource    agendaBackend
+	offChainSource   politeiaBackend
```

Comment:
Please update on/off everywhere.  It just does not have enough meaning to be clear.  Someone reading the code has to look back to the `explorerUI` to see what they are really for (agendas and Pi).